src/main/resources/com/puppycrawl/tools/checkstyle/checks/design/messages_zh.properties

```diff
@@ -1,10 +1,9 @@
-arrangement.members.before.inner=初始化块，构造函数，字段和方法应位于内部类型之前。
-design.forExtension=類 ''{0}'' 看起來像是為擴展設計的（可以是子類），但方法 ''{1}'' 沒有javadoc，解釋瞭如何安全地執行。\
-  如果類不是為擴展而設計的，請考慮創建類 ''{0}'' final或使方法 ''{1}'' static/final/abstract/empty，或為方法添加允許的
+arrangement.members.before.inner=内部方法/内部接口的定义应位于初始化代码块、构造方法、成员变量和方法的定义之后。
+design.forExtension=既然类 ''{0}'' 设计为可继承的，应为可重写的非空方法 ''{1}'' 添加 javadoc 文档说明。
```

Comment:
Since class ''{0}'' is designed to be inheritable, add javadoc documentation for overridable non-null method ''{1}''.